src/util/agent_control.c

```diff
@@ -605,15 +607,58 @@ int main(int argc, char **argv)
     /* Run active response on the specified agent id */
     if (ip_address && ar && (agent_id || restart_all_agents)) {
         /* Connect to remoted */
+        FILE *fp;
+        int pass= 0;
         mdebug1("Connecting to remoted...");
         arq = connect_to_remoted();
         if (arq < 0) {
             printf("\n** Unable to connect to remoted.\n");
             exit(1);
         }
         mdebug1("Connected...");
+        fp = fopen(DEFAULTAR, "r");
+        if (fp) {
+            char buffer[256];
+
+            while (fgets(buffer, 255, fp) != NULL) {
```

Comment:
Soft parser: if doesn't enforce the file integrity.
A line like `restart-ossec0- restart-ossec.sh - 0` would pass.